http-troubleshooting.md

```diff
@@ -0,0 +1,37 @@
+# Kubecost HTTP Response Code Troubleshooting
+
+If you have experienced an HTTP response code while using Kubecost, consult this doc for a possible fix.
+
+You will likely encounter these response codes in the browser, when performing REST requests through a client like cURL, or when viewing the logs `kubectl logs deploy/kubecost-cost-analyzer -c cost-analyzer-frontend -n kubecost`.
+
+## HTTP 403 Forbidden
+
+This is most likely due to a user authenticating to Kubecost or performing an action in Kubecost that they don't have permissions to do (for example, saving a report when the user has read-only access). It can be caused by mesh networking, firewalls, load balancers, or some other service 
```

Comment:
There is lower likelihood of this error being caused by "mesh networking, firewalls, load balancers, or some other service"

It could be safer to make the following statement:
> "There is also a possibility that this error is caused by intermediary services such as load balancers, firewalls, or service meshes"